### ISSUE TITLE
DO_WHILE task is shown as a single box on new UI

### ISSUE THREAD

**Issue by astelmashenko** [Netflix/conductor#2725]
**Describe the bug**
Since new UI is out, it does not show DO_WHILE task list, it is just a single box. Before it was expanded and DO_WHILE task list was shown.

**Details**
Conductor version: 3.4.1
Workflow definition:
```
{
  "name": "WaitTest",
  "description": "",
  "ownerApp": "test",
  "ownerEmail": "test@test.io",
  "schemaVersion": 2,
  "version": 2,
  "tasks": [
{
      "name": "LoopTask",
      "taskReferenceName": "LoopTask0",
      "type": "DO_WHILE",
      "inputParameters": {
        "value": "Success"
      },
      "loopCondition": "if ( $.value == $.wait_test.response.body.result) { false; } else { true; }",
      "loopOver": [
        {
            "name": "wait_test",
            "taskReferenceName": "wait_test_ref",
            "type": "WAIT"
        }
      ]
    }
  ]
}
```

**To Reproduce**
1. submit workflow definition
2. observe workflow on new UI

**Expected behavior**
LoopTask shows it's internal task list

**Screenshots**
![Screenshot from 2022-01-27 15-19-29](https://user-images.githubusercontent.com/2878309/151366901-51d70193-27c0-4be2-833c-70b52aac4b7c.png)

**Comment by manan164** [comment]
Hi @astelmashenko , I see there is a fix for this. [loop ui](https://github.com/Netflix/conductor/pull/1824). Can you please check whether this is included in 3.4.1 or not?

**Comment by astelmashenko** [comment]
@manan164 , nope, if you look at #1824 fix, it is for _old_ UI, it is included into v2.29.0. Now there is new UI implementation without that fix, unfortunately.

**Comment by peterlau** [comment]
A implementation of the visualization of DO_WHILE is in the works. @boney9 can provide more details.

**Comment by boney9** [comment]
I'll submit a PR for this by this week. Thanks for pointing this out @peterlau

**Comment by aravindanr** [comment]
DO_WHILE task visualization is available in 3.8.1 and greater..
